operations/outletitems.md

```diff
@@ -41,120 +44,118 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+| `OutletBills` | array of [Outlet bill](outletitems.md#outlet-bill) | optional | The outlet bills of the items. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest outlet item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older outlet items. |
 
 #### Outlet item
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the item. |
 | `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
-| `BillId` | string | required | Unique identifier of the [Outlet bill](#outlet-bill) the item belongs to. |
+| `BillId` | string | required | Unique identifier of the [Outlet bill](outletitems.md#outlet-bill) the item belongs to. |
 | `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](accountingcategories.md#accounting-category) the item belongs to. |
-| `Type` | string [Outlet item type](#outlet-item-type) | required | Type of the item. |
-| `Name` | string | required | Name of the item. |
-| `UnitCount` | number | required | Unit count of the item. |
-| `UnitAmount` | [Amount parameters](orders.md#amount-parameters) | required | Unit amount of the item. |
-| `CreatedUtc` | string | optional | Date and time of the item creation in UTC timezone in ISO 8601 format. |
-| `ConsumedUtc` | string | required | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
+| `Type` | string | required | Type of the item. |
+| `Name` | string | optional | Name of the item. |
+| `UnitCount` | integer | required | Unit count of the item. |
+| `UnitAmount` | [Amount](_objects.md#amount) | required | Unit amount of the item. |
+| `CreatedUtc` | string | required | Date and time of the item creation in UTC timezone in ISO 8601 format. |
 | `UpdatedUtc` | string | required | Last update date and time of the item in UTC timezone in ISO 8601 format. |
+| `ConsumedUtc` | string | required | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
 | `ExternalIdentifier` | string | optional | An identifier of this item from another system. |
 | `Notes` | string | optional | Additional notes. |
-| `PaymentCardPaymentId` | string | optional | Unique identifier of the payment card [Payment](payments.md#payment) this item is linked to. This is only applicable to items of [Type](#outlet-item-type) `Payment`. |
-
-
-#### Outlet item type
-
-* `Revenue`
-* `NonRevenue`
-* `Payment`
+| `PaymentCardPaymentId` | string | optional | Unique identifier of the payment card [Payment](#payment) this item is linked to. This is only applicable to items of [Type](#outlet-item-type) `Payment`. |
+| ~~`UnitCost`~~ | ~~[Currency value (ver 2018-06-07)](_objects.md#currency-value-ver-2018-06-07)~~ | ~~optional~~ | ~~Total price of the reservation.~~ **Deprecated!** |
```

Comment:
issue: There should be a recommendation on what other property to use; I think `UnitAmount` is the recommended alternative; so in the monolith you'll put this on the UnitCost property:

```
[Deprecated("Use `UnitAmount` instead.")]
```